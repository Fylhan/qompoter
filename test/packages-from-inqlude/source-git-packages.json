{
	"name": "packages-from-inqlude/source-git-packages",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"kde/attica": "v5.25.0",
		"kde/bluez-qt": "v5.24.*",
		"cp/fylhan": "v1.1"
	},
	"require-dev": {
		"laumaya/glc-lib": "dev-master"
	}
}
