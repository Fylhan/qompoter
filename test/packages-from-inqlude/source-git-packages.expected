Qompoter
======== install -> create qompoter.lock

* kde/attica v5.25.0
  Downloading sources from Git...
  Warning: no 'qompoter.pri' found for this package
  done

* kde/bluez-qt v5.24.*
  Downloading sources from Git...
  Selected version: v5.24.0
  Warning: no 'qompoter.pri' found for this package
  done

* cp/fylhan v1.1
  Downloading sources...
  done

* laumaya/glc-lib dev-master
  Warning: there may be no such inqlude package as dev-master, try v2.5.2 if it fails
  Downloading sources from Git...
  Warning: no 'qompoter.pri' found for this package
  done

done
